tdrop

```diff
@@ -118,6 +121,7 @@ do
 	f) program_flags=$OPTARG;;
 	a) auto_detect_wm=true;;
 	m) monitor_aware=true;;
+	t) pointer_screen_detection=true;;
 	-)
 		if [[ $OPTARG =~ ^(auto-detect-wm|monitor-aware|clear|no-cancel|help)$ ]] || \
```

Comment:
Please add `pointer-screen-detection` to this line (all flags that don't take an argument need to be in this check).